kimchi/src/circuits/polynomials/generic.rs

```diff
@@ -207,7 +207,7 @@ impl<F: FftField + SquareRootField> ConstraintSystem<F> {
 
             // alpha
             let alpha_pow = {
-                let mut res = self.precomputations.constant_1_d4.clone();
+                let mut res = self.precomputations.get().unwrap().constant_1_d4.clone();
```

Comment:
I'm not sure how I feel about this `unwrap()`. The nice thing about OnceCell is that you can evaluate the value the first time you get it, and then cache it for subsequent calls.

I think it'd be nicer to have a `self.precomputations()` that never fails